egs/swbd/s5c/local/nnet3/run_reduced_frame.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+# This scripts does nnet3-tdnn training and decoding 
+# (just like nnet3/run_tdnn.sh) except that it uses chain topology and
+# a frame subsampling factor of 3
+
+# At this script level we don't support not running on GPU, as it would be painfully slow.
+# If you want to run without GPU you'd have to call train_tdnn.sh with --gpu false,
+# --num-threads 16 and --minibatch-size 128.
+
+stage=0
+affix=
+train_stage=-10
+has_fisher=true
+speed_perturb=true
+common_egs_dir=
+reporting_email=
+remove_egs=true
+leftmost_questions_truncate=-1
+
+. ./cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+
+if ! cuda-compiled; then
+  cat <<EOF && exit 1
+This script is intended to be used with GPUs but you have not compiled Kaldi with CUDA
+If you want to use GPUs (and have them), go to src/, and configure and make on a machine
+where "nvcc" is installed.
+EOF
+fi
+
+suffix=
+if [ "$speed_perturb" == "true" ]; then
+  suffix=_sp
+fi
+dir=exp/nnet3/reduced_tdnn
+dir=$dir${affix:+_$affix}
+dir=${dir}$suffix
+train_set=train_nodup$suffix
+ali_dir=exp/tri4_ali_nodup$suffix
+treedir=exp/nnet3/reduced_tdnn_tree$suffix
+lang=data/lang_reduced
+
+local/nnet3/run_ivector_common.sh --stage $stage \
+	--speed-perturb $speed_perturb || exit 1;
+
+
+if [ $stage -le 9 ]; then
+  # Create a version of the lang/ directory that has one state per phone in the
+  # topo file. [note, it really has two states.. the first one is only repeated
+  # once, the second one has zero or more repeats.]
+  rm -rf $lang
+  cp -r data/lang $lang
+  silphonelist=$(cat $lang/phones/silence.csl) || exit 1;
+  nonsilphonelist=$(cat $lang/phones/nonsilence.csl) || exit 1;
+  # Use our special topology... note that later on may have to tune this
+  # topology.
+  steps/nnet3/chain/gen_topo.py $nonsilphonelist $silphonelist >$lang/topo
```

Comment:
While trying this non-special first state, I got this error in build-tree.sh:

> cluster-phones exp/nnet3/tdnn_lfr2a_tree_sp/treeacc data/lang_lfr2a/phones/sets.int exp/nnet3/tdnn_lfr2a_tree_sp/questions.int 
> WARNING (cluster-phones:AutomaticallyObtainQuestions():build-tree.cc:576) After filtering the tree statistics to retain only stats where pdf-class is in the set { 1 }, most of your stats disappeared: the size changed from 69459 to 0.  You might be using a nonstandard topology but forgot to modify the --pdf-class-list option (it defaults to { 1 } which is the central state in a 3-state left-to-right topology). E.g. a 1-state HMM topology would require the option --pdf-class-list=0.
> ERROR (cluster-phones:EnsureClusterableVectorNotNull():cluster-utils.cc:94) All stats are NULL.

Should I give the --pdf-class-list option to cluster-phones or something else is wrong?